### PR TITLE
Add a test for an incorrect .dockstore.yml property

### PR DESCRIPTION
**Pull request by wshands**
Addresses:
#3564 
https://ucsc-cgl.atlassian.net/browse/DOCK-1418

### CODE REVIEW

**Comment by wshands** [comment]
@coverbeck I am not sure what else to do about this ticket; the error message seems sort of OK.

**Comment by coverbeck** [comment]
I put in a comment in the issue about this could be improved. Up to @denis-yuen if 

https://github.com/dockstore/dockstore/issues/3564#issuecomment-646300154

File: dockstore-common/src/test/java/io/dockstore/common/yaml/DockstoreYamlTest.java [hunk 1]
```diff
@@ -182,4 +183,22 @@ public void testDifferentCaseForWorkflowSubclass() throws DockstoreYamlHelper.Do
...
+        // Replace:
+        // ...
+        // workflows:
+        //
+        // With incorrect:
+        // ...
+        // workflow:
+        final String content = DOCKSTORE12_YAML.replaceFirst("workflows", "workflow");
+        try {
+            DockstoreYamlHelper.readAsDockstoreYaml12(content);
```
Comment (denis-yuen) [inline dockstore-common/src/test/java/io/dockstore/common/yaml/DockstoreYamlTest.java:198]: This test won't quite work if it throws some other exception or just doesn't throw an exception

Comment (wshands) [reply dockstore-common/src/test/java/io/dockstore/common/yaml/DockstoreYamlTest.java:199]: Fixed up test

Comment (denis-yuen) [reply dockstore-common/src/test/java/io/dockstore/common/yaml/DockstoreYamlTest.java:199]: Won't this still "succeed" with some other exception, say an IOException?

Comment (wshands) [reply dockstore-common/src/test/java/io/dockstore/common/yaml/DockstoreYamlTest.java:199]: Now will catch any other exception and fail the test

File: dockstore-common/src/main/java/io/dockstore/common/yaml/DockstoreYamlHelper.java [hunk 2]
```diff
@@ -21,6 +22,7 @@
 import org.yaml.snakeyaml.constructor.Constructor;
 import org.yaml.snakeyaml.introspector.Property;
 import org.yaml.snakeyaml.introspector.PropertyUtils;
+import scala.tools.nsc.transform.patmat.Logic;
```
Comment (denis-yuen) [inline dockstore-common/src/main/java/io/dockstore/common/yaml/DockstoreYamlHelper.java:25]: additional imports seem a bit superfluous if nothing else is changing in this file

**Comment by denis-yuen** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/dockstore/dockstore/pullRequest?prid=6277226&bid=20431336)

**Comment by denis-yuen** [comment]
@wshands  status of this PR?

**Comment by denis-yuen** [comment]
Seems addressed by different PR in review, re-open if incorrect

**Pull request closed without merging**